package.json

```diff
@@ -4,7 +4,17 @@
   "version": "1.1.3",
   "author": "Austin Green",
   "dependencies": {
+    "@ckeditor/ckeditor5-basic-styles": "^18.0.0",
```

Comment:
These packages don't appear to be used. Let's remove all of the unused ones (everything except react-rte), then delete the yarn.lock file and rerun `yarn`